crates/rpc/rpc-builder/src/lib.rs

```diff
@@ -1705,6 +1705,50 @@ impl TransportRpcModules {
         self.merge_ipc(other)?;
         Ok(())
     }
+
+    /// Removes the method with the given name from the configured http methods.
+    ///
+    /// Returns `true` if the method was found and removed, `false` otherwise.
+    pub fn remove_http_method(&mut self, method_name: &'static str) -> bool {
```

Comment:
these need the disclaimer from `remove_method`, we can reference this function [`RpcModule::remove_method`]

> Be aware that a subscription consist of two methods, subscribe and unsubscribe and it's the caller responsibility to remove both subscribe and unsubscribe methods for subscriptions.

Reply:
Makes sense, will add!

Reply:
Resolved in 93a510aa620cb50134a9275842b95a3ffbe6aa9f